src/Microsoft.AspNet.Identity/SignInManager.cs

```diff
@@ -53,24 +56,17 @@ public class SignInManager<TUser> where TUser : class
         internal IUserClaimsPrincipalFactory<TUser> ClaimsFactory { get; private set; }
         internal IdentityOptions Options { get; private set; }
         internal ILogger Logger { get; set; }
+        protected IdentityLogger IdentityLogger { get; set; }
```

Comment:
I changed it to internal and I feel it makes sense to have it protected. In the constructor of the subclass, I can set the Identity logger passing the logger for the subclass which is then reflected in all the log statements, else we have log statements with base class and the sub class which looked weird to me